typescript/scripts/README.adoc

```diff
@@ -0,0 +1,65 @@
+:toc: macro
+
+= tBTC v2 Refund script
+
+This script explains what to do in case you need to refund your BTC.
+
+toc::[]
+
+== Refund script
+
+=== Prerequisites
+
+Please make sure you have the following prerequisites installed on your machine:
+
+- https://nodejs.org[Node.js] >14.18.0
+- https://yarnpkg.com[Yarn] >1.22.19
+
+tBTCv2 system prerequisite is that you can refund your BTC only if it was not used 
+for minting tBTC and after `refundLocktime` has passed.
+In practice, you should wait about an hour longer than `refundLocktime` since the 
+transaction is only allowed to enter the Bitcoin blockchain if the median time of 
+the last 11 blocks is equal to or greater than `refundLocktime`.
+Currently the `refundLocktime` is set to 9 months.
+
+=== Running a refund script
+
+Step 1. Clone https://github.com/keep-network/tbtc-v2[tbtc-v2] repository to your 
+local disc. `git clone https://github.com/keep-network/tbtc-v2`
+
+Step 2. Navigate to a root directory and run the following command with your arguments:
+```
+./typescript/scripts/refund.sh 
+    --deposit-json-path <deposit-JSON-file-path> \
+    --deposit-transaction-amount <deposit-transaction-amount> \
+    --deposit-transaction-id <deposit-transaction-id> \
+    --deposit-transaction-index <deposit-transaction-index> \
+    --private-key <recoverer-private-key> \
+    --transaction-fee <transaction-fee> \
+    --electrum-host <electrum-host> \
+    --electrum-port <electrum-port> \
+    --electrum-protocol <electrum-protocol>
+```
+Every argument must be provided but electrum related info. You can set you own 
+electrum connection if for some reason the default one doesn't work. However, the 
+following arguments must be provided:
+
+- `deposit-JSON-file-path` Path to a file that you downloaded during the BTC 
+deposit in the Threshold dApp.
+- `deposit-amount` Exact BTC amount in satoshi that was deposited. It will not process a 
+partial refund.
+- `deposit-transaction-id` A transaction ID of your BTC deposit.
+- `deposit-transaction-index` A transaction index of your deposit UTXO that can 
+be retrieved from a bitcoin blockchain explorer. If it's first on the list then 
+the index is 0, if it's second then the index is 1, etc.
+- `private-key` A private key of the BTC recovery wallet. If you only have a 
+passphrase, one of the options is to extract your private keys by using 
+https://iancoleman.io/bip39/[iancoleman] website (can be extracted offline) or 
```

Comment:
I wouldn't specify any particular online solution, since we have not guarantee the website is safe.